src/Json.Schema/JsonSchema.cs

```diff
@@ -108,7 +108,9 @@ public JsonSchema(JsonSchema other)
             MinLength = other.MinLength;
             Format = other.Format;
             MultipleOf = other.MultipleOf;
+            Minimum = other.Minimum;
```

Comment:
>Minimum = other.Minimum; [](http://example.com/codeflow?start=12&length=24)

This is a existing bug, it is not set.
So I can not use Max + Min for the `auto`

Reply:
Ouch. So we clearly do not have sufficient tests. For this kind of testing, I typically use advanced reflection techniques to instantiate object instances. I'd populate them with random values. Then I'd test the ctor here and the basic equatable functionality. Something to keep in mind if we have time.

Reply:
Created a follow-up issue in the other comment.